service/src/main/java/com/ge/predix/acs/attribute/connector/management/AttributeConnectorController.java

```diff
@@ -15,24 +15,42 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 import com.ge.predix.acs.commons.web.AcsApiUriTemplates;
 import com.ge.predix.acs.commons.web.BaseRestApi;
 import com.ge.predix.acs.commons.web.RestApiException;
 import com.ge.predix.acs.rest.AttributeConnector;
 
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;;
+
+
 @RestController
 public class AttributeConnectorController extends BaseRestApi {
 
     @Autowired
     private AttributeConnectorServiceImpl service;
 
+    @ApiOperation(
+            value = "Creates or updates connector configururation for external resource attributes for the given zone.",
+            tags = { "Attribute Connector Management" },
+            response = AttributeConnector.class)
+    @ApiResponses(
+            value = { @ApiResponse(
+                    code = 201,
```

Comment:
I couldn't find a way to display a description for the default response status.  Default status is always displayed at the top as "Response Class (Status 200)" or "Response Class (Status 204)" if I change the default to 204.

Version of swagger we use is 2.2.2.  May be it's possible in a later versions.